src/AbstractPlayerBridge.ts

```diff
@@ -57,27 +68,90 @@ export abstract class AbstractPlayerBridge extends LiveryBridge {
     if (name === 'getEndpointId') {
       return this.getEndpointId();
     }
+    if (name === 'getFeatures') {
+      return this.getFeatures();
+    }
     if (name === 'getLatency') {
       return this.getLatency();
     }
     if (name === 'getLiveryParams') {
       return this.getLiveryParams();
     }
+    if (name === 'getPlayback') {
+      return this.getPlayback();
+    }
     if (name === 'getPlayerVersion') {
       return this.getPlayerVersion();
     }
     if (name === 'getStreamId') {
       return this.getStreamId();
     }
+    if (name === 'pause') {
+      return this.pause();
+    }
+    if (name === 'play') {
+      return this.play();
+    }
+    if (name === 'reload') {
+      return this.reload();
+    }
+    if (name === 'seek') {
+      return this.seek(arg as number);
```

Comment:
I can pass them as unknown, but then I would also have to set the method parameter's definition as unknown as well. Then in the video player repo when implementing the method it will be prefilled with the unknown type instead of the defined one. I thought this way provides better documentation for what the bridge expects.

Reply:
Ah, yes, it would be better if the abstract methods just work with proper typed arguments. We should just handle all type checking here in the AbstractPlayerBridge. So far I did (intend to) do full type checking here though to fail early and clearly when a player bridge implementation posts invalid messages.

Since it was not much before I didn't bother, but like I said I wouldn't mind adding Zod or something like that now to help facilitate all of this runtime type checking. So feel free to do that if you like. Otherwise we can just keep on writing type checking boiler plate and perhaps a few utility methods for that purposes. Whatever you prefer.

So, I think we should now implement and use something like `validateCommandName()` methods here in the AbstractPlayerBridge that return validated method arguments for the concrete or abstract command implementation methods?